### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by dionyziz**

### CODE REVIEW

**Comment by dionyziz** [comment]
Fixes #57

File: controllers/testrun.php [hunk 1]
```diff
@@ -21,10 +24,20 @@ public function create( $name, $all = false ) {
...
+                        $failed = true;
+                    }
+                }
 
                 $unittests[] = $unittest;
             }
             require_once 'views/testrun/results.php';
+
+            if ( $failed ) {
+                return 1;
```
Comment (pkakelas) [inline controllers/testrun.php:38]: You should use true;

File: controllers/testrun.php [hunk 2]
```diff
@@ -21,10 +24,20 @@ public function create( $name, $all = false ) {
...
+            }
+            return 0;
```
Comment (pkakelas) [inline controllers/testrun.php:40]: false;

File: views/testrun/results.txt.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<?php
+    echo "\n\n";
+    echo "================\n";
+    echo pluralize( count( $unittests ), 'unit test' ) . "\n";
+    echo "================\n\n";
+?>
```
Comment (pkakelas) [inline views/testrun/results.txt.php]: Why do you close the ?> and in the next line you open it again?

File: controllers/testrun.php (hunk 2 above)
Comment (dionyziz) [inline controllers/testrun.php:40]: No - this is a shell [exit status](http://tldp.org/LDP/abs/html/exit-status.html) and must be a byte numeral (from 0 to 255).

Comment (pkakelas) [inline controllers/testrun.php:40]: Ok then.

**Comment by dionyziz** [comment]
CLI for test failure:

image

**Comment by dionyziz** [comment]
CLI for test success:

image

**Pull request merged**